src/routes/news.js

```diff
@@ -33,4 +33,46 @@ router.post('/', (req, res) => {
 		.catch(err => res.status(500).json({error: err}))
 });
 
+router.get('/:news_id', (req, res) => {
+	News
+		.findById(req.params.news_id)
+		.then(news => {
+			res.status(200).json(news)
+		})
+		.catch(err => res.status(500).send({error: err}))
+});
+
+router.put('/:news_id', (req, res) => {
+	const { title, text } = req.body;
+	let errors = [];
+
+	if (!title || title.length > TITLE_MAX_LENGTH) errors.push(`title should contain 1 - ${TITLE_MAX_LENGTH} characters`);
+	if (!text || text.length > TEXT_MAX_LENGTH) errors.push(`text should contain 1 - ${TEXT_MAX_LENGTH} characters`);
+
+	if (errors.length) return res.status(400).json({errors: errors});
+
+	News
+		.update({
+			"title": req.body.title,
+			"text": req.body.text,
+			"updatedAt": new Date()
+		},
+		{
+			returning: true, where: {id: req.params.news_id}
+		})
+		.then(news => {
+			res.status(202).json(news)
```

Comment:
why 202?
202: The request has been accepted for processing, but the processing has not been completed. The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place.
204 i think